subprojects/tooling-api/build.gradle.kts

```diff
@@ -6,6 +6,10 @@ plugins {
 
 gradlebuildJava.usedInToolingApi()
 
+tasks.named<Jar>("sourcesJar") {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Do we have duplicate sources here? If we have, can you add a comment which ones?